include/power/ndsx_espi.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __NDSX_ESPI_H__
+#define __NDSX_ESPI_H__
+
+/**
+ * @brief Configure eSPI channels for Non Deep Slwwp well platforms
+ */
+void ndsx_espi_configure(void);
+
+/**
+ * @brief Return Virtual wire value
+
+ * @retval Acatual Virtual Wire value or 0 on failure
```

Comment:
s/Acatual/Actual/

Should we return an int and use a negative return to indicate an error instead?

Reply:
ACK, will change it